tests/TestModel.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Spatie\Backup\Test;
+
+use Illuminate\Database\Eloquent\Model;
+
+class TestModel extends Model
```

Comment:
This model is used to add sample data to the temporary sqlite db. This can be removed if needed since the backup dump would contain at least the table creating queries even without sample rows.